lib/features.js

```diff
@@ -51,6 +52,18 @@ module.exports = config => {
     },
 
     /**
+     * Return whether or not secondary email support is enabled.
+     *
+     * @returns {boolean}
+     */
+    isSecondaryEmailEnabled() {
+      if (! secondaryEmail.enabled) {
+        return false
+      }
+      return true
```

Comment:
This is quite an impressive line count for what looks like `return secondaryEmail.enabled` (or at a push, `!!`). :smile: